docs/rfcs/002-archival_storage/README.md

```diff
@@ -290,15 +290,16 @@ Content-Type: application/json
 When storing an update, the service will:
 
 - Check that the supplied version matches the current version
-- Validate and unpack the supplied bag
+- Unpack the supplied bag
 - Store the supplied bag as a new version
-- Update the current version of the bag
+- Register the new version of the bag as the current version
 
-Partial updates, where files that are not changed are not resupplied, are supported through the use of `fetch.txt` in the supplied bag. File references must include the bag version in which the file was previously supplied.
+Partial updates, where files that are not changed are not resupplied, are supported through the use of `fetch.txt` in the supplied bag. File references must specify the full storage location of the previously supplied file, including the version number of the bag in which it was last supplied in the path.
```

Comment:
Perhaps an example `fetch.txt`
> https://s3-eu-west-1.amazonaws.com/wellcomecollection-storage-staging-access/alex-testing/b12345/v3/data/b24923333.xml 98765 data/b24923333.xml

Reply:
There's one in the partial bag example later on in the RFC.